PureBasicIDE/dialogs/Preferences.xml

```diff
@@ -49,6 +49,7 @@
                   <checkbox id="#GADGET_Preferences_MemorizeWindow"       lang="Preferences:MemorizeWindow" />
                   <checkbox id="#GADGET_Preferences_AutoReload"           lang="Preferences:AutoReload" />
                   <checkbox id="#GADGET_Preferences_DisplayFullPath"      lang="Preferences:DisplayFullPath" />                
+                  <checkbox id="#GADGET_Preferences_DisplayDarkMode"      lang="Preferences:DisplayDarkMode" />                
```

Comment:
If the settings only has a meaning for macOS then the checkbox should be hidden (or removed) for the other OS. Otherwise we have an option which does absolutely nothing which is very frustrating for a user.

Maybe something like "Dark Mode" can be implemented for the other OS as well?

Reply:
You don't have to hide the option, but you could lock the option for other OS. Like with options that are only for Windows.